scripts/sendpayment.py

```diff
@@ -75,7 +77,23 @@ def main():
                 sys.exit(EXIT_ARGERROR)
             destaddr = parsed['address']
             if 'jmnick' in parsed:
+                if "pj" in parsed:
+                    parser.error("Cannot specify both BIP79++ and Joinmarket "
```

Comment:
BIP79++ should be changed to BIP78 here and in a comment below.

Reply:
I've been working on the BIP78 changes (after a lot of comments, the BIP appears to be stable now), and I've already changed the comments there. Thanks.

Reply:
Addressed in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/536/commits/41540ab53e4ae86ffdd02688e689e56df3356229